### PR TITLE
Correct typos in examples

### PR DESCRIPTION
**Pull request by ron1**

### CODE REVIEW

File: multi-cluster/kustomize/kustomization.yaml [hunk 1]
```diff
@@ -1,2 +0,0 @@
-resources:
-- overlays/dev
```
Comment (StrongMonkey) [inline multi-cluster/kustomize/kustomization.yaml:2]: Why is this removed?

File: single-cluster/helm-multi-chart/guestbook/README.md [hunk 2]
```diff
@@ -1,17 +0,0 @@
-# Helm Example
```
Comment (StrongMonkey) [inline single-cluster/helm-multi-chart/guestbook/README.md:1]: Why is this removed also?

Comment (ron1) [reply single-cluster/helm-multi-chart/guestbook/README.md:1]: This patch adds the README.md for this example to the parent `helm-multi-chart` directory. The GitRepo in the deleted README.md references paths `single-cluster/helm` which is incorrect. Also, having two README.md files, one in the `helm-multi-chart` directory and one in the `helm-multi-chart/guestbook` directory seems confusing to me.

File: multi-cluster/kustomize/kustomization.yaml (hunk 1 above)
Comment (ron1) [reply multi-cluster/kustomize/kustomization.yaml:2]: The root `kustomization.yaml` here is confusing to me. Is its intent to make the `dev` overlay the default?` Since there are three environments, I expected kustomize users of this repo to explicitly name the target environment to ensure changes are not accidentally applied to the wrong cluster. Defaulting to the `dev` environment seems dangerous to me. If my current context happens to be a cluster other than `dev` and I do not know that overlays exist in this repo for multiple environments, I could easily `kustomize build | kubectl apply -f -` to the wrong cluster.

Comment (StrongMonkey) [reply multi-cluster/kustomize/kustomization.yaml:2]: @ron1 I think the original intent was to provide a working example without user tweaking too much. After you removed, I would expect the example to fail unless user adds a kustomization.yaml? I feel like you can just add the comment on this file says that the context(overlay/dev) needs to be changed across different environments.

Comment (ron1) [reply multi-cluster/kustomize/kustomization.yaml:2]: The typical kustomize build flow for variants with overlays is available [here](https://github.com/kubernetes-sigs/kustomize#2-create-variants-using-overlays). Note there is no `kustomization.yaml` in the top-level directory. Instead, the following overlay-specific command is used in the build flow as described at that link: `kustomize build overlays/<env>. So no additional `kustomization.yaml` must be added in this case to make the example work. Instead an additional overlay parameter is simply added to the `kustomize build` command. Again the canonical kustomize build flow for variants with overlays referenced above demonstrates this behavior.

**Review by StrongMonkey — APPROVED** [review]
LGTM

**Comment by ibrokethecloud** [comment]
LGTM
